contrib/zenith/libpagestore.c

```diff
@@ -34,23 +34,23 @@ void		_PG_init(void);
 
 #define PqPageStoreTrace DEBUG5
```

Comment:
Shouldn't we rename this to PageStoreTrace instead of PqPageStoreTrace, considering that the library was renamed from libpqpagestore to libpagestore?